### ISSUE TITLE
Accessibility Insights: listview - Focusable content should have tabindex='-1' or be removed from the DOM

### ISSUE THREAD

**Issue by RomanVakhterov** [winjs/winjs#1699]
Title: WCAG 4.1.2,WCAG 1.3.1: ARIA hidden element must not contain focusable elements (div[aria-hidden="true"]:nth-child(1))
Tags: Accessibility, WCAG 4.1.2, WCAG 1.3.1, aria-hidden-focus​
​
Issue: ARIA hidden element must not contain focusable elements (aria-hidden-focus - https://dequeuniversity.com/rules/axe/3.3/aria-hidden-focus?application=msftAI)​
​
Element path: div[aria-hidden="true"]:nth-child(1)​
​
Snippet: `​<div tabindex="0" aria-hidden="true"></div>`
​
How to fix: ​
Fix all of the following:​
  Focusable content should have tabindex='-1' or be removed from the DOM​
​
